docker-app/qfieldcloud/core/models.py

```diff
@@ -126,6 +126,20 @@ class UserAccount(models.Model):
         (TYPE_PRO, "pro"),
     )
 
+    # Keep these in sync with qfieldcloud.settings.CRON_CLASSES and qfieldcloud.core.cron
+    NOTIF_DISABLED = "DISABLED"
+    NOTIF_IMMEDIATELY = "IMMEDIATELY"
+    NOTIF_HOURLY = "HOURLY"
+    NOTIF_DAILY = "DAILY"
+    NOTIF_WEEKLY = "WEEKLY"
+    NOTIF_CHOICES = (
+        (NOTIF_IMMEDIATELY, _("Immediately")),
+        (NOTIF_HOURLY, _("Hourly")),
+        (NOTIF_DAILY, _("Daily")),
+        (NOTIF_WEEKLY, _("Weekly")),
+        (NOTIF_DISABLED, _("Disabled")),
+    )
+
```

Comment:
Can we use `models.TextChoices` instead? For me it's cleaner, rather that a bunch of capitalized properties. Also better for typing.